packages/near-sdk-js/src/cli/cli.ts

```diff
@@ -85,6 +86,21 @@ program
       .option("--verbose", "Whether to print more verbose output.", false)
       .action(transpileJsAndBuildWasmCom)
   )
+  .addCommand(
+    new Command("watchGas")
+      .usage("[target] [methodName params...] [methodName params...]")
+      .description(
+        "Measure gas used for a contract method. Run this command after deploying the contract."
+      )
+      .argument(
+        "[target]",
+        "Target file path and name. (e.g., ./build/counter.wasm)"
+      )
+      .argument(
+        "[...methodCalls]", "Method calls with their parameters (e.g., increase n=5 decrease n=2)"
+      )
+      .action(measureGas)
```

Comment:
How is it different from using near.cli.rs and watching for "﻿Gas burned" in the output?

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/a49b4851-fd07-4d49-a872-319114278690">

```sh
$ near contract call-function as-transaction qq.frol8.testnet f1 json-args '{}' prepaid-gas '100.0 Tgas' attached-deposit '0 NEAR'
```

Reply:
All transactions here are executed on sandbox env. The idea of this command was to make it easier for new people to start working with near-sdk-js.

Reply:
Hey @frol, should we close this PR?